src/main/java/com/example/bookstore/service/impl/BookSeviceImpl.java

```diff
@@ -40,4 +40,14 @@ public BookDto getBookById(Long id) {
                 .toDto(byId.orElseThrow(() ->
                         new EntityNotFoundException("cant find book by id")));
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public void updateById(Long id, CreateBookRequestDto createBookRequestDto) {
+        bookRepository.updateBookById(id, createBookRequestDto);
```

Comment:
Let's use default `bookRepository.save()` instead of custom updateBookById. It will be simpler.
Also, add check if updated entity record is present.